src/authors/authors.service.ts

```
@@ -1,80 +1,74 @@
+        }
+        return author;
     }
-    return author;
-  }
 
-  async updateAuthor(id: string, params: UpdateAuthorInput) {
-    const bookIds =
-      params.bookIds && params.bookIds.map((id) => new Types.ObjectId(id));
-    delete params['bookIds'];
+    async updateAuthor(id: string, params: UpdateAuthorInput): Promise<AuthorDocument> {
+        const bookIds = params.bookIds && params.bookIds.map(id => new Types.ObjectId(id));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)